### PR TITLE
[new release] spotify_ml (0.0.2)

### PR DESCRIPTION
**Pull request by maxRN**
Spotify API bindings

- Project page: <a href="https://github.com/maxrn/spotify_ml">https://github.com/maxrn/spotify_ml</a>

##### CHANGES:

- add documentation
- add proper error types

### CODE REVIEW

**Comment by mseri** [comment]
Thanks

File: packages/spotify_ml/spotify_ml.0.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "Spotify API bindings"
+description: "Interact with the Spotify Web API via OCaml."
+maintainer: ["maxrn"]
```
Comment (mseri) [inline packages/spotify_ml/spotify_ml.0.0.2/opam:4]:
```suggestion
maintainer: ["maxrn"]
```
It would be useful to add a way to contact you here

Comment (maxRN) [reply packages/spotify_ml/spotify_ml.0.0.2/opam]: Thanks, updated.

File: packages/spotify_ml/spotify_ml.0.0.2/opam [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+maintainer: ["Max Große <max12.gro@gmail.com>"]
+authors: ["Max Große <max12.gro@gmail.com>"]
+license: "MIT"
+tags: ["spotify"]
+homepage: "https://github.com/maxrn/spotify_ml"
+bug-reports: "https://github.com/maxrn/spotify_ml/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.16"}
+  "ocamlformat" {dev & = "0.26.2"}
```
[the remark this answers is not in the document]
Comment (maxRN) [reply packages/spotify_ml/spotify_ml.0.0.2/opam]: Yes thank you. I didn't know about this one, because it's not available in dune

**Comment by maxRN** [comment]
Why does the CI keep saying `[spotify_ml.0.0.2 (failed: Sorry, resolution of the request timed out.)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/d42d7a5a85c40e7caa76d582bd6e8f535aa2f3d1/variant/extras,opam-2.0-ocaml-5.2,spotify_ml.0.0.2)`? Can I do anything to change that? @mseri

**Comment by mseri** [comment]
Thanks

**Pull request merged**